# Default initial positions for fetch's ros2_control fake system

initial_positions:
  elbow_flex_joint:  1.7784
  forearm_roll_joint: 0
  l_gripper_finger_joint: 0
  r_gripper_finger_joint: 0
  shoulder_lift_joint: 1.5
  shoulder_pan_joint:  1.59
  upperarm_roll_joint: 0
  wrist_flex_joint:  1.1814
  wrist_roll_joint: -1.57
